spec/factories/keywords.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```

Comment:
Setting up factory girl is alright, but please try to not have empty factories set up!